Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_IOCs_-_Disable_expired_IOCs_in_XDR.yml

```diff
@@ -0,0 +1,908 @@
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "10"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 490,
+          "y": 20
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: 3060f6ad-7512-4dd1-861b-2bef3a325725
+    type: regular
+    task:
+      id: 3060f6ad-7512-4dd1-861b-2bef3a325725
+      version: -1
+      name: Set Done to True
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "26"
+    scriptarguments:
+      append:
+        simple: "false"
+      key:
+        simple: Done
+      value:
+        simple: "True"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1770,
+          "y": 980
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: b566fb1a-5750-4ec6-8ce4-c96d09ffe889
+    type: regular
+    task:
+      id: b566fb1a-5750-4ec6-8ce4-c96d09ffe889
+      version: -1
+      name: Add xdr_disabled tag
+      description: Appends values to the specified indicator field. Supported field types are tags and multi-select.
+      script: Builtin|||appendIndicatorField
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "14"
+    scriptarguments:
+      field:
+        simple: tags
+      fieldValue:
+        simple: xdr_disabled
+      indicatorsValues:
+        simple: ${vetted_iocs}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 510,
+          "y": 2075
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: eaf0fad0-2942-4e4b-8e81-0198128680e4
+    type: regular
+    task:
+      id: eaf0fad0-2942-4e4b-8e81-0198128680e4
+      version: -1
+      name: Get Indicators
+      description: Gets a list of indicator objects and the associated indicator outputs that match the specified query and filters. The results are returned in a structured data file.
+      scriptName: GetIndicatorsByQuery
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "11"
+    scriptarguments:
+      limit:
+        simple: ${inputs.batch_size}
+      populateFields:
+        simple: value,sha256,md5,indicator_type
```

Comment:
you dont need md5 since XDR only works with sha256

Reply:
_Define the DATA FORMAT of the IOCs in the CSV file. Options are Mixed, Full Path, File Name, Domain, Destination IP, and MD5 or SHA256 Hash._

https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Create-an-IOC-Rule